runtime/ops/process.rs

```diff
@@ -227,6 +227,20 @@ fn create_command(
   command.args(args.args);
 
   if let Some(cwd) = args.cwd {
+    if let Ok(cwd_metadata) = std::fs::metadata(cwd.clone()) {
+      if !cwd_metadata.is_dir() {
+        return Err::<std::process::Command, _>(AnyError::msg(format!(
```

Comment:
When adding a test, we should ensure that it still maintains the existing error class. I believe it's `Deno.errors.NotFound` (I think this could be maintained by instead just adding to the error message of the `std::io::Error`, but I'm not sure).